code/datums/mind.dm

```diff
@@ -75,12 +75,15 @@
 	src.key = key
 
 /datum/mind/proc/transfer_to(mob/new_character)
+	if(!new_character)
+		return
 	if (!current)
 		transfer_to_without_current(new_character)
 		return
-
-	new_character.attack_log += current.attack_log
-	new_character.attack_log += "\[[time_stamp()]\]: mind transfer from [current] to [new_character]"
+	if(new_character.attack_log && current.attack_log)
```

Comment:
this will make the admins job harder, though neither the old code nor this version are good